src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -107,7 +107,7 @@ private function __construct(array $thresholds = [], $regex = '', $verboseOutput
             if (!isset($this->verboseOutput[$group])) {
                 throw new \InvalidArgumentException(sprintf('Unsupported verbosity group "%s", expected one of "%s".', $group, implode('", "', array_keys($this->verboseOutput))));
             }
-            $this->verboseOutput[$group] = (bool) $status;
+            $this->verboseOutput[$group] = $status;
```

Comment:
`@param bool[]      $verboseOutput` in the doc + the constructor is private and only called with a `$verboseOutput` in `fromUrlEncodedString()` and `inWeakMode()` and we always set valid booleans values.